conf/modules.config

```diff
@@ -861,23 +872,129 @@ process {
     withName: ".*BBMAP_BBDUK" {
         tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
         ext.args = { "entropymask=f entropy=${params.metagenomics_complexity_entropy}" }
-        ext.prefix = { "${meta.sample_id}_${meta.library_id}_complexity" }
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}_complexity" }
         publishDir = [
-            path: { "${params.outdir}/metagenomic_complexity_filter/" },
+            path: { "${params.outdir}/metagenomics/complexity_filter/bbduk/" },
             mode: params.publish_dir_mode,
             pattern: '*.{fastq.gz,log}',
             enabled: params.metagenomics_complexity_savefastq
         ]
     }
 
-    //
-    // MT-NUCLEAR RATIO
-    //
-    withName: MTNUCRATIO {
-        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+    withName: MALT_RUN {
+        ext.args = [
+            "-m ${params.metagenomics_malt_mode}",
+            "-at ${params.metagenomics_malt_alignmentmode}",
+            "-top ${params.metagenomics_malt_toppercent}",
+            "-id ${params.metagenomics_malt_minpercentidentity}",
+            "-mq ${params.metagenomics_malt_maxqueries}",
+            "--memoryMode ${params.metagenomics_malt_memorymode}",
+            params.metagenomics_malt_minsupportmode == "percent" ? "-supp ${params.metagenomics_malt_minsupportpercent}" : "-sup ${params.metagenomics_minsupportreads}",
+            params.metagenomics_malt_savereads ? "--alignments ./" : ""
+        ].join(' ').trim()
         publishDir = [
-            enabled: false
+            path: { "${params.outdir}/metagenomics/profiling/malt/" },
+            mode: params.publish_dir_mode,
+            pattern: '*.{rma6,log,sam.gz}'
+        ]
+        ext.prefix = { "${meta.label}_${meta.id}-run" }
+    }
+
+    withName: CAT_CAT_MALT {
+        ext.prefix = { "${meta.id}_runtime_log_concatenated.log" }
+        publishDir = [
+            path: { "${params.outdir}/metagenomics/profiling/malt/" },
+            mode: params.publish_dir_mode,
+            pattern: '*.{log}'
+        ]
+    }
+
+    withName: KRAKEN2_KRAKEN2 {
+        ext.args = [
+            params.metagenomics_kraken2_saveminimizers ? "--report-minimizer-data" : ""
```

Comment:
Are there no other parameters someone may want to use?

Reply:
move to new PR